src/ansi-c/c_typecheck_code.cpp

```diff
@@ -836,3 +837,21 @@ void c_typecheck_baset::typecheck_spec_decreases(codet &code)
     }
   }
 }
+
+void c_typecheck_baset::disallow_function_history_variables(
```

Comment:
Why do we need this? There is `c_typecheck_baset::disallow_history_variables` defined in `c_typecheck_expr.cpp`